`azurerm_kubernetes_cluster` - support for the `multiple_standard_load_balancers_enabled` property (#21072)
hashicorp/terraform-provider-azurerm: website/docs/r/kubernetes_cluster.html.markdown

~> **Note:** `managed_outbound_ipv6_count` requires dual-stack networking. To enable dual-stack networking the Preview Feature `Microsoft.ContainerService/AKS-EnableDualStack` needs to be enabled and the Resource Provider re-registered, see [the documentation](https://docs.microsoft.com/azure/aks/configure-kubenet-dual-stack?tabs=azure-cli%2Ckubectl#register-the-aks-enabledualstack-preview-feature) for more information.

> * `multiple_standard_load_balancer_enabled` - (Optional) Is multiple standard load balancers per AKS cluster enabled?

Reviewer: grammer
```suggestion
* `multiple_standard_load_balancer_enabled` - (Optional) Are multiple standard load balancers for the cluster enabled?
```
